WordPress/Classes/Services/SignupService.swift

```diff
@@ -1,84 +1,74 @@
 import Foundation
 import CocoaLumberjack
 import WordPressShared
+import WordPressKit
 
 
 
-typealias SignupSocialSuccessBlock = (_ newAccount: Bool) -> Void
-typealias SignupFailureBlock = (_ error: Error?) -> Void
-
-
-/// SignupService is responsible for creating a new WPCom user and blog.
-///
-open class SignupService: LocalCoreDataService {
+// MARK: - SignupService: Responsible for creating a new WPCom user and blog.
+//
+class SignupService {
 
     /// Create a new WPcom account using Google signin token
     ///
     /// - Parameters:
-    ///   - token: the token from a successful Google login
+    ///   - googleToken: the token from a successful Google login
     ///   - success: block called when account is created successfully
     ///   - failure: block called when account creation fails
-    func createWPComUserWithGoogle(token: String,
-                                   success: @escaping SignupSocialSuccessBlock,
-                                   failure: @escaping SignupFailureBlock) {
+    ///
+    func createWPComUser(googleToken: String,
```

Comment:
Personally I like `createWPComUserWithGoogle` better 😜 

Either way, renaming this method doesn't seem critical to decoupling. It'd make reviews easier if I didn't have to consider so many changes 😬 also, if we're renaming things changing `SignupService` to mention `Google` would make a lot of sense (as mentioned in #9109)

Reply:
Swift style would probably be...

```
func createWPComUser(with googleToken:
```

But you're right, this isn't a requirement for decoupling. I was in the neighborhood and felt like improving things a bit.

For the sake of shrinking this PR, i'll revert this change.

[Reference here](https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage).